### ISSUE TITLE
Create vmdk o qcow2

### ISSUE THREAD

**Issue by hkfuertes** [MatthewCroughan/raspberrypi-nixos-example#1]
How difficult would it be to use the same `configuration.nix` file to make a vm hdd? Say I want to fully test the rpi image that I have before writting it to sd and so on... can I run something like `nix build .#images.vm` to create a virt-manager compatible hdd on my x64 laptop?

**Comment by MatthewCroughan** [comment]
Why do you want to do that? Why not just use NixOS VM tests?

**Comment by hkfuertes** [comment]
True!... just yesterday discover this rabbit hole... and a vmdk is something I understand, hahahaha, I will investigate the nixos vms.... for now im not using nixos, but just nix under mint... Im trying it out, but looks promising, specially to replace my docker devenvs (specially for android/flutter development, as it is easier for android sdk to access real hardware)

Thank you for your quick response
